README.md

```diff
@@ -1,18 +1,29 @@
 # Twilio Voice React Native Reference
 
-## What is this Project?
+This project consists of a backend server and a react native app demonstrating best practices for developing a Twilio Programmable Voice application.
 
-The Twilio Voice React Native Reference App is an example implementation of the Twilio Voice React Native SDK and serves to inspire developers who want to leverage the power of Twilio Programmable Voice in their React Native applications.
+## Features
 
-* See the Twilio Voice React Native SDK [here](https://github.com/twilio/twilio-voice-react-native).
+* Dialpad for making outgoing calls
+  * Call to a phone number
+  * Call to a Twilio Client<sup>*</sup>
+* Active call screen to interact with and control ongoing calls
+* Login flow for authenticating and authorizing users<sup>*</sup>
+* Registration and notifications for incoming calls<sup>*</sup>
 
-## Before you use this Project
+<sup>*</sup> Planned for a future release.
 
-You should have the React Native development environment set up and have a firm understanding of Twilio Programmable Voice.
+![dialer](https://user-images.githubusercontent.com/22135968/223575482-53f733b4-c828-4dd1-b100-7cc945a52c9f.jpg)
+![active](https://user-images.githubusercontent.com/22135968/223575472-f112ab20-f11c-4f54-9432-3640ddbb0f1c.jpg)
 
-Please follow the official React Native guides to set up your environment, and additionally the Twilio Programmable Voice documentation. TODO LINKS
+## Prerequisites
 
-### Structure
+- Understanding of [Twilio Programmable Voice](https://www.twilio.com/docs/voice/sdks)
+- [React Native Development Environment](https://reactnative.dev/docs/environment-setup)
```

Comment:
```suggestion
- A [React Native Development Environment](https://reactnative.dev/docs/environment-setup).
```
Should we note the version of RN we use and link to those docs, or at least recommend users follow the setup guides for that version?

Reply:
Good idea. I'll use the previous link you used.